dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -115,6 +115,18 @@
       )
     %p= t('user.only_one_parent_email_supported_warning')
 
+- if experiment_value('ai_rubrics', request)
+  - if current_user.verified_teacher?
```

Comment:
Sorry if I missed this but why only verified teachers?

Reply:
I don't have a strong opinion here so whatever you think is best! Was just curious as to how we came to that decision

Reply:
We should revisit this question! I think we were again considering verified teachers only for now. @bethanyaconnor should we do this check here, too?

Reply:
I'll also tag @tess323 here but, yeah, I think it makes sense to check for verified teacher permissions here